### PR TITLE
Github actions

### LINKED ISSUE #10: Replace travis by Github Actions

### PR DESCRIPTION
**Pull request by ErickZH**
- Now the laravel discounts support laravel 5.8 or above

Fixed #10

### CODE REVIEW

**Comment by zaratedev** [comment]
Thanks @ErickZH

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: laravel-discounts
+        ports:
+          - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+
```
Comment (zaratedev) [inline .github/workflows/tests.yml:34]: Remove line

File: README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ $item = Item::find(1);
 $item->createDiscount(100);
 
 ```
+## Requirements
+The Laravel Discounts package requires **PHP 7.2** or above and **Laravel 5.8** or above.
```
Comment (zaratedev) [inline README.md]: The Laravel Discounts package requires **PHP >= 7.2** and **Laravel >= 5.8**.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (ErickZH) [reply .github/workflows/tests.yml:34]: Ready

**Review by zaratedev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ErickZH) [reply README.md]: Ready

**Review by zaratedev — APPROVED** [review]

**Pull request merged**